### PR TITLE
offscreencanvas: Enabled by default in Ch69

### PR DESCRIPTION
**Pull request by Tyriar**
https://www.chromestatus.com/feature/5681560598609920

### CODE REVIEW

File: features-json/offscreencanvas.json [hunk 1]
```diff
@@ -276,7 +276,9 @@
       "4.2-4.3":"n",
       "4.4":"n",
       "4.4.3-4.4.4":"n",
-      "67":"n"
+      "67":"n",
+      "68":"n",
```
Comment (HM100) [inline features-json/offscreencanvas.json]: No new browser version can be added via PR. Please drop that line

File: features-json/offscreencanvas.json [hunk 2]
```diff
@@ -276,7 +276,9 @@
...
+      "69":"y"
```
Comment (HM100) [inline features-json/offscreencanvas.json]: No new browser version can be added via PR. Please drop that line

File: features-json/offscreencanvas.json (hunk 1 above)
Comment (HM100) [inline features-json/offscreencanvas.json]: And remove the comma from that line as well

File: features-json/offscreencanvas.json [hunk 3]
```diff
@@ -276,7 +276,9 @@
       "4.2-4.3":"n",
       "4.4":"n",
       "4.4.3-4.4.4":"n",
-      "67":"n"
+      "67":"n",
```
Comment (HM100) [inline features-json/offscreencanvas.json]: And remove the comma from that line as well

**Comment by Tyriar** [comment]
Done

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**